clientv3/integration/kv_test.go

```diff
@@ -59,9 +59,9 @@ func TestKVPutError(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	time.Sleep(500 * time.Millisecond) // give enough time for commit
+	time.Sleep(1 * time.Second) // give enough time for commit
 
-	_, err = kv.Put(ctx, "foo2", strings.Repeat("a", int(maxReqBytes-50)))
+	_, err = kv.Put(ctx, "foo2", strings.Repeat("a", int(quota-50)))
```

Comment:
I think this can stay `maxReqBytes-50` since `2*(maxReqBytes-50) > (maxReqBytes + 8*pagesize)`